build/export-variables.yml

```diff
@@ -0,0 +1,9 @@
+variables:
+    ResourceGroupRegion: 'southcentralus'
+    resourceGroupRoot: 'msh-fhir-export'
+    appServicePlanName: '$(resourceGroupRoot)-linux'
+    DeploymentEnvironmentName: '$(resourceGroupRoot)'
+    ResourceGroupName: '$(resourceGroupRoot)'
+    TestEnvironmentName: 'OSS Export'
+    ImageTag: 'master'
+    nuGetVersion: '5.8.1'
```

Comment:
Where is this variable used?

Reply:
This is used in `package.yml`

Reply:
Ah, this is one of the variables from Semver. Why don't we use the update-semver script to get this variable like in the ci-pipeline?

Reply:
I initially had it as part of the pipeline. But I removed it since we aren't really going to use that version information anywhere else. Hence decided to use a hardcoded variable for nuget version. In the future I would like to remove the build step completely and download the artifacts from the CI pipeline (using the `DownloadPipelineArtifact` task). But there are a couple of other changes that need to be done before we can do that.